.github/workflows/reusable-build.yml

```diff
@@ -113,7 +113,7 @@ jobs:
       # Fix: Resolve disk space error "ENOSPC: no space left on device" on GitHub Actions runners
       - name: Free disk cache
         if: ${{ startsWith(runner.name, 'GitHub Actions') && inputs.target == 'x86_64-unknown-linux-gnu' && steps.check_cache.outputs.exists != 'true' && !inputs.skipable }}
-        uses: jlumbroso/free-disk-space@main
+        uses: xc2/free-disk-space@main
```

Comment:
I prefer pin to a release tag instead of `main`, `main` is unstable

Reply:
DONE